cli/liblisa-semantics-tool/src/server.rs

```diff
@@ -247,11 +247,13 @@ impl Server {
                     bincode::deserialize_from(BufReader::new(file)).unwrap()
                 },
                 Err(e) => {
-                    eprintln!("Error reading {cache:?}: {e}");
+                    match e.kind() {
+                        ErrorKind::NotFound => eprintln!("Creating cache {cache:?}"),
+                        _ => eprintln!("Error reading {cache:?}: {e}"),
+                    } 
 
                     let map = build_filter_map(&self.encodings);
 
-                    eprintln!("Writing cache to {cache:?}");
```

Comment:
I removed this to avoid having both "Creating cache XXX" and "Writing cache to XXX" as that would also be confusing.

Unfortunately this means that in the case where there is an error reading the file the code now prints the error and moves on to create the cache anyways. Possibly overwriting another file.

I don't know what the desired behavior is in this case, but I wanted to get this submitted without letting the perfect be the enemy of the good enough.